doc/api/tls.markdown

```diff
@@ -202,63 +202,64 @@ established it will be forwarded here.
 
 `function (sessionId, sessionData, callback) { }`
 
-Emitted on creation of TLS session. May be used to store sessions in external
+Emitted on creation of a TLS session. May be used to store sessions in external
```

Comment:
For consistency, we might use _"This event is emitted"_ like we did for the `'secure'` event.